src/book/entities/Price.ts

```diff
@@ -94,6 +98,64 @@ export default class Price extends BaseEntity {
   get id(): string {
     return `${this.date.toISODate()}.${this.commodity.mnemonic}.${this.currency.mnemonic}`;
   }
+
+  async save(): Promise<this> {
+    await Price.upsert(
+      [this],
+      {
+        conflictPaths: ['fk_commodity', 'fk_currency', 'date'],
+      },
+    );
```

Comment:
instead of having to use `upsert` in forms or other parts where we have to remember that, now we just make sure that any save creates an upsert